components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/AuthenticationContext.java

```diff
@@ -180,6 +180,25 @@ public void setConsumerKey(String consumerKey) {
         this.consumerKey = consumerKey;
     }
 
+    private int applicationSpikesArrestLimit;
```

Comment:
Shall we add the variables along with the other variables on the top?

Reply:
Refactored in https://github.com/wso2/carbon-apimgt/pull/12162/commits/547e395745eb20cdba4453df74dd33ff5a91ac7c